bin/www

```diff
@@ -4,39 +4,18 @@
  * Module dependencies.
  */
 
-var app = require('../app');
-var debug = require('debug')('simul-doc-be:server');
-var http = require('http');
-
-/**
- * Get port from environment and store in Express.
- */
-
-var port = normalizePort(process.env.PORT || '3000');
-app.set('port', port);
-
-/**
- * Create HTTP server.
- */
-
-var server = http.createServer(app);
-
-/**
- * Listen on provided port, on all network interfaces.
- */
-
-server.listen(port);
-server.on('error', onError);
-server.on('listening', onListening);
+const app = require('../app');
+const debug = require('debug')('simul-doc-be:server');
+const http = require('http');
 
 /**
  * Normalize a port into a number, string, or false.
  */
 
 function normalizePort(val) {
-  var port = parseInt(val, 10);
+  const port = parseInt(val, 10);
```

Comment:
You can convert string to number by using `Number('123')` function. This function also accepts number so you don't need to check later if it's not a number.

Reply:
I guess it's not your code :)

Reply:
Yes, it was autogenerated.